src/commands.rs

```diff
@@ -105,6 +104,8 @@ enum Commands {
     Backout(BackoutArgs),
     #[command(subcommand)]
     Branch(BranchSubcommand),
+    #[command(subcommand)]
+    Conflict(ConflictSubcommand),
```

Comment:
I think it would be good to keep `jj resolve` as an alias for `jj conflict resolve`, since this is a command that people will probably run quite frequently. See how `undo` and `op undo` are done if you want an example.

Reply:
I'm beginning to think that maybe `jj conflict resolve` wasn't such a great idea. If you want `jj resolve` to keep working, we can just have that, at least at first. Listing conflicts could be `jj resolve --list`. WDYT, @martinvonz

Reply:
SGTM